src/extension.ts

```diff
@@ -24,6 +69,28 @@ export async function activate(context: vscode.ExtensionContext) {
                                      context.workspaceState);
       }));
 
+  // Create a command to open the project root .clangd configuration file.
+  context.subscriptions.push(
+      vscode.commands.registerCommand('clangd.projectSettings', () => {
+        if (vscode.workspace.workspaceFolders) {
+          const folder = vscode.workspace.workspaceFolders[0];
```

Comment:
Hmm, how sure are we that this is the right directory (that workspace folders correspond to the real project root, and folder 0 is the right one?)

Reply:
There's unfortunately not a great way of going about this. In a single workspace mode this should be sufficient. But once you start working with multi folder workspaces things start getting a little hazy. However a lot of projects users are likely to work on only have 1 root folder so this should be helpful for the majority of users.